scripts/20newsgroups-replication.py

```diff
@@ -19,43 +21,44 @@ def get_info(path):
     return (docs, targets)
 
 
+class VectorizerType(Enum):
+    TFIDF = 'tfidf'
+    BM25 = 'bm25'
+
+
 if __name__ == '__main__':
-    target_names = [
-        'alt.atheism',
-        'comp.graphics',
-        'comp.os.ms-windows.misc',
-        'comp.sys.ibm.pc.hardware',
-        'comp.sys.mac.hardware',
-        'comp.windows.x',
-        'misc.forsale',
-        'rec.autos',
-        'rec.motorcycles',
-        'rec.sport.baseball',
-        'rec.sport.hockey',
-        'sci.crypt',
-        'sci.electronics',
-        'sci.med',
-        'sci.space',
-        'soc.religion.christian',
-        'talk.politics.guns',
-        'talk.politics.mideast',
-        'talk.politics.misc',
-        'talk.religion.misc',
-    ]
+    parser = argparse.ArgumentParser(
+        description='Replication script of pyserini vectorizer')
+    parser.add_argument('-vectorizer', type=VectorizerType, required=True, help='which vectorizer to use')
```

Comment:
`--vectorizer`? per what we agreed to? https://github.com/lintool/guide/blob/master/coding-style.md